docs/en/wallet-instance-attestation.rst

```diff
@@ -2,56 +2,676 @@
+   :name: sequence diagram for wallet instance attestation request
+   :alt: The figure shows the sequence diagram for issuing a wallet instance attestation.
+         The steps will be described below.
+
+`PlantUML <https://www.plantuml.com/plantuml/uml/XPB1RjH038RlynIc9v1OSU-XAk9GWSG50RtqucH-HLOJJ_1u6csVdcbsTqG85LVxy-VypjncP_CoZU7DR3nCJ8xqJ6u5WOidBLC72s6kbFGoipfT_SYmA-9CPLk_vt64qsUjKksn8elya-cuVuJ64zA5KEXmKzdhEYmkFCepQ3cXSjOzQ38o_2h8_c4sP5HVRuZGbuaS5ZdSBDqrtS4lixEb9uamck0SsJaitQ5ITT7NSuNwfCwYeiCVDlBZZFoU7d5STufXWdgjGEESHFsyhvf-yYZLXDrIjvATHibUsKl0EoYiqgjwPh4SGbDzChoq_ZeaVSmPvfAKlftoqvVxxu5xbwTrRxV9per--r_HktgGTNANGYup0xI8Gf7p7iuoA7injDPmoSSQr_PEsBwl_OpN0--F_BejOdkHwYvDtNXf3om-g87ZaJnHwfn5IR5idjGjD9Pf_0q0>`__
+
+- **Message 1**: The end user initializes the wallet instance.
+  In particular, this process happens when the mobile application
+  is launched and every time the end user wants to request or present
+  (disclose) a credential.
+- **Message 2-3**: The wallet instance obtains metadata about its wallet
+  provider. Among these, we also find the list of supported algorithms,
+  public keys, endpoints.
+- **Message 4**: The wallet instance verifies that the wallet provider is
+  trustworthy by resolving the provider's trust chain up to the anchor (⚠️
+  this step is skipped in this version)
+- **Message 5-7**: The wallet instance instantiates a new key pair on its TEE
+  and requests a nonce from the backend wallet (as a measure against replay
+  attacks)
+- **Message 8**: The wallet instance generates a Wallet Instance Attestation
+  Request (JWT) signed with the private key associated with the public key
+  for which it wants to obtain the attestation containing the nonce and
+  other useful parameters.
+- **Message 9-13**: The wallet instance sends the Wallet Instance Attestation
+  Request to the backend wallet which verifies its validity and issues the
+  signed attestation.
+- **Message 13-14**:The wallet instance receives the wallet instance
+  attestation signed by the wallet provider and proceeds with a formal
+  verification
+- **Message 15**:The wallet instance attestation is ready to be consumed
+
+
+Detail design
+---------------
+
+Format of the Wallet Instance Attestation Request
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To obtain a Wallet Instance Attestation from the wallet
+provider (backend) it is necessary to send a Wallet Instance Attestation
+Request from the wallet instance containing the associated public key
+and a nonce previously requested to avoid replay attacks.
+
+Header
+^^^^^^
++-----------------------------------+-----------------------------------+
```

Comment:
does this render well in sphinx using rst?

if not, please use the template we have for tables

Reply:
yes, it is compatible. Furthermore, the table was generated with the docs italia [tool](https://docs.italia.it/converti/)